packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md

```diff
@@ -9,24 +9,26 @@ meta:
 
 This document explains how to configure <StackSelector snippet="idp" noSelector inline /> as an external Identity Provider for your application by creating an application at <StackSelector snippet="idp" noSelector inline />, creating an Identity Provider in Okta, testing the configuration, and creating a sign-in button.
 
-Okta manages the connection to the IdP for your application, sitting between your application and the IdP that authenticates your users. The industry-standard term for this is Inbound Federation. When a user signs in, you can link the user’s Identity Provider account to an existing Okta user profile or choose to create a new user profile using Just-In-Time (JIT) provisioning.
-
-> **Note:** We also support additional services such as directories and credential providers. See the [Okta Integration Network Catalog](https://www.okta.com/okta-integration-network/) to browse all integrations by use case.
-
 ---
 
 **Learning outcomes**
 
-How to configure an external Identity Provider so that your users can quickly sign up or sign in to your application using their social Identity Provider account.
+Configure an external Identity Provider so that your users can quickly sign up or sign in to your application using their social Identity Provider account.
```

Comment:
```suggestion
Configure an external Identity Provider so that your users can quickly sign up or sign in to your app using their social Identity Provider account.
```